lib/src/uptech_growthbook_wrapper.dart

```diff
@@ -27,10 +27,25 @@ class UptechGrowthBookWrapper {
   /// Initialize for use in app, seeds allow you to specify value of
   /// toggles prior to fetching remote toggle states. These will be
   /// the values if on init it fails to fetch the toggles from the remote.
-  Future<void> init(
-      {Map<String, dynamic>? seeds,
-      Map<String, dynamic>? overrides,
-      Map<String, dynamic>? attributes}) async {
+  Future<void> init({
+    // Updates to the growthbook_sdk_flutter package changed the structure of features
+    // This workaround allows seeded features to still be passed in using the old format,
+    // as well as features using the new format with GBFeature. We simply convert the
+    // seeds into GBFeatures and add them to the final Map<String, GBFeature>.
+    Map<String, GBFeature> features = const <String, GBFeature>{},
+    @Deprecated('Instead use "features" as Map<String, GBFeature>')
+        Map<String, dynamic> seeds = const <String, dynamic>{},
+    Map<String, dynamic>? overrides,
+    Map<String, dynamic>? attributes,
+  }) async {
+    Map<String, GBFeature> finalFeatures = Map.from(features);
+    // Convert seeds to GBFeatures and add to finalFeatures Map
+    if (seeds.isNotEmpty) {
+      seeds.forEach((key, value) {
+        finalFeatures[key] = GBFeature(defaultValue: value);
+      });
```

Comment:
I don't see any tests that ensure that `seeds` are still valid, and that they populate growthbook correctly (because they are merged here). Should they be added?